docroot/modules/custom/sitenow_articles/sitenow_articles.module

```diff
@@ -556,6 +573,12 @@ function sitenow_articles_preprocess_block(&$variables) {
 
       break;
 
+    // @todo Move this to a more appropriate spot, if we may
+    //   want to use the gallery image field for other content.
+    case 'field_block:node:article:field_gallery_images':
+      $variables['attributes']['class'][] = 'grid--fivecol--20 photoswipe-gallery';
```

Comment:
Yep--it wouldn't run into quite the same issue that brought it to our attention since it's at least within an array, but for consistency and possibly other issues, it should be split apart.